gtd-timer/Startup.cs

```diff
@@ -16,6 +16,9 @@
 using gtdtimer.Timer.DAL.Entities;
```

Comment:
for the using formatting, let's agree on following:

- always use 'Remove and sort usings' VS command from context menu
- put 'System' usings first followed by new line
The whole thing should look like this: 

using System.Reflection;
using System.IO;

using gtdtimer.Timer.DAL;
...